lib/diagnostic.js

```diff
@@ -8,15 +8,18 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-const min = function min() {
-
+const min = function min(array) {
+  array.reduce(function(prev, curr) {
+  });
```

Comment:
You misunderstood this question; we were asking you to create a function, `min`, that we could pass into `reduce` as a callback argument.
i.e.
```
[1,2,3].reduce(min);
// => 1
```